.github/workflows/build-docker-image.yml

```diff
@@ -32,33 +32,51 @@ jobs:
       - name: go-test
         run: go test -v ./...
 
-  build:
+  build-image-for-linux:
     runs-on: ubuntu-latest
     needs: test
     steps:
-    - uses: actions/checkout@v3
-    - uses: actions/setup-go@v5
-      with:
-        go-version: '1.21'
-    - run: go version
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '1.21'
```

Comment:
Let's use the version from the go.mod file: https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file